### PR TITLE
[4.3] License.txt

### PR DESCRIPTION
**Pull request by brianteeman**
Spotted during a license compliance check that some files did not have the correct information

code review

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: libraries/src/Event/Plugin/System/Webauthn/Ajax.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * Joomla! Content Management System
  *
  * @copyright  (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
- * @license    General Public License version 2 or later; see LICENSE
+ * @license    General Public License version 2 or later; see LICENSE.txt
```
Comment (heelc29) [inline libraries/src/Event/Plugin/System/Webauthn/Ajax.php]:
```suggestion
 * @license    GNU General Public License version 2 or later; see LICENSE.txt
```

File: libraries/src/Event/Plugin/System/Webauthn/AjaxChallenge.php [hunk 2]
```diff
@@ -4,7 +4,7 @@
  * Joomla! Content Management System
  *
  * @copyright  (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
- * @license    General Public License version 2 or later; see LICENSE
+ * @license    General Public License version 2 or later; see LICENSE.txt
```
Comment (heelc29) [inline libraries/src/Event/Plugin/System/Webauthn/AjaxChallenge.php]:
```suggestion
 * @license    GNU General Public License version 2 or later; see LICENSE.txt
```

File: libraries/src/Event/Plugin/System/Webauthn/AjaxCreate.php [hunk 3]
```diff
@@ -4,7 +4,7 @@
  * Joomla! Content Management System
  *
  * @copyright  (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
- * @license    General Public License version 2 or later; see LICENSE
+ * @license    General Public License version 2 or later; see LICENSE.txt
```
Comment (heelc29) [inline libraries/src/Event/Plugin/System/Webauthn/AjaxCreate.php]:
```suggestion
 * @license    GNU General Public License version 2 or later; see LICENSE.txt
```

File: libraries/src/Event/Plugin/System/Webauthn/AjaxDelete.php [hunk 4]
```diff
@@ -4,7 +4,7 @@
  * Joomla! Content Management System
  *
  * @copyright  (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
- * @license    General Public License version 2 or later; see LICENSE
+ * @license    General Public License version 2 or later; see LICENSE.txt
```
Comment (heelc29) [inline libraries/src/Event/Plugin/System/Webauthn/AjaxDelete.php]:
```suggestion
 * @license    GNU General Public License version 2 or later; see LICENSE.txt
```

File: libraries/src/Event/Plugin/System/Webauthn/AjaxInitCreate.php [hunk 5]
```diff
@@ -4,7 +4,7 @@
  * Joomla! Content Management System
  *
  * @copyright  (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
- * @license    General Public License version 2 or later; see LICENSE
+ * @license    General Public License version 2 or later; see LICENSE.txt
```
Comment (heelc29) [inline libraries/src/Event/Plugin/System/Webauthn/AjaxInitCreate.php]:
```suggestion
 * @license    GNU General Public License version 2 or later; see LICENSE.txt
```

File: libraries/src/Event/Plugin/System/Webauthn/AjaxLogin.php [hunk 6]
```diff
@@ -4,7 +4,7 @@
  * Joomla! Content Management System
  *
  * @copyright  (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
- * @license    General Public License version 2 or later; see LICENSE
+ * @license    General Public License version 2 or later; see LICENSE.txt
```
Comment (heelc29) [inline libraries/src/Event/Plugin/System/Webauthn/AjaxLogin.php]:
```suggestion
 * @license    GNU General Public License version 2 or later; see LICENSE.txt
```

File: libraries/src/Event/Plugin/System/Webauthn/AjaxSaveLabel.php [hunk 7]
```diff
@@ -4,7 +4,7 @@
  * Joomla! Content Management System
  *
  * @copyright  (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
- * @license    General Public License version 2 or later; see LICENSE
+ * @license    General Public License version 2 or later; see LICENSE.txt
```
Comment (heelc29) [inline libraries/src/Event/Plugin/System/Webauthn/AjaxSaveLabel.php]:
```suggestion
 * @license    GNU General Public License version 2 or later; see LICENSE.txt
```

**Comment by heelc29** [comment]
I have tested this item :white_check_mark: successfully on 857e051c9655d905560a24a63457cf062e1aca9b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40400">issues.joomla.org/tracker/joomla-cms/40400</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 857e051c9655d905560a24a63457cf062e1aca9b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40400">issues.joomla.org/tracker/joomla-cms/40400</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40400">issues.joomla.org/tracker/joomla-cms/40400</a>.</sub>

**Comment by obuisard** [comment]
Thanks for the 'clean up' Brian @brianteeman !

**Comment by brianteeman** [comment]
thanks

**Pull request merged**